src/cli.ts

```diff
@@ -241,7 +249,7 @@ export async function run (process: NodeJS.Process): Promise<number> {
       flattenComponents: options.flattenComponents,
       shortPURLs: options.shortPURLs
     },
-    myConsole
+    logger
```

Comment:
now that it is possible, a child logger with a custom  `msgPrefix= 'BomBuilder > '`  might be used here.
see https://github.com/pinojs/pino/blob/master/docs/api.md#optionsmsgprefix-string

Reply:
added child logger